app/next-client-app/types/scanreport.ts

```diff
@@ -14,8 +14,9 @@ interface ScanReportTable {
   updated_at: Date;
   name: string;
   scan_report: number;
-  person_id: string;
-  date_event: string;
+  person_id: string | null;
+  date_event: string | null;
+  permissions: string[];
```

Comment:
All the permissions here - as before, should be of the type `Permission[]`

https://github.com/Health-Informatics-UoN/CaRROT-Mapper/blob/8d9b83936c438415caabc1073306261c50e57fdb/app/next-client-app/types/permissions.ts